react/src/AddDatasets/index.tsx

```diff
@@ -52,6 +52,13 @@ export default function AddParticipants(props: {
         }
     }, [props.groups]);
 
+    function onChangeGroups(newGroups: string[]) {
+        setAsGroups(newGroups);
+        if (newGroups.length === 0) {
+            setErrorMessage("Cannot submit. You must select a permission group.");
+        }
```

Comment:
I think the error condition needs to consider both the filled-out state of the entire form and the dropdown. This worked initially, but when I filled out the table, the submit button was enabled again.